frontend/components/molecules/DropdownFilters/index.tsx

```diff
@@ -39,7 +39,7 @@ type Props = {
 
 const FilterTexts: FilterTextsType = {
     DATE: { 1: 'Newest first', 2: 'Oldest first' },
-    ANSWER: { 1: 'Answered', 2: 'Unanswered' },
+    ANSWER: { 1: 'Answered', 2: 'Unanswered', 3: 'All Answers' },
```

Comment:
Maybe a better name for showing all questions is `All Questions` or `Show All` or just `All`